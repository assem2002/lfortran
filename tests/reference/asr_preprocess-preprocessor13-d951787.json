{
    "basename": "asr_preprocess-preprocessor13-d951787",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor13.f90",
    "infile_hash": "a846ad5320eb7082135106ba078c94cc21b41c4365d735839474f3ed",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor13-d951787.stdout",
    "stdout_hash": "f891e891cc4fd86b67574fb17d6abb40ef0ddfed6ba6408a44a94775",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}