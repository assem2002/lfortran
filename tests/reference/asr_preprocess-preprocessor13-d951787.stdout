(TranslationUnit
    (SymbolTable
        1
        {
            __xx_main:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    __xx_main
                    []
                    [(Print
                        [(StringConstant
                            "1 LFortran"
                            (Character 1 10 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "2 LFortran 1"
                            (Character 1 12 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "3 LFortran 1"
                            (Character 1 12 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "4 LFortran 1"
                            (Character 1 12 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "42 LFortran 1"
                            (Character 1 13 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X is 1"
                            (Character 1 6 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "1 Y is 1"
                            (Character 1 8 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X is 2"
                            (Character 1 6 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "2 Y is 3"
                            (Character 1 8 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X is 4"
                            (Character 1 6 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "4 Y is 4"
                            (Character 1 8 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "X is not 1-4"
                            (Character 1 12 ())
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringConstant
                            "5 Y is not 1-4"
                            (Character 1 14 ())
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
